### PR TITLE
Clarifies where the IPFS icon is on menubar

### PR DESCRIPTION
**Pull request by Annamarie2019**
I'd like to add an image.

### CODE REVIEW

**Comment by bertrandfalguiere** [comment]
Isn't that just valid for MacOS?

File: docs/install/ipfs-desktop.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ If you already have an IPFS node on your computer, IPFS Desktop will act as a co
 
 ![Status screen of IPFS Desktop](./images/ipfs-desktop/desktop-status.png)
 
-| Files screen                                                               | Explore screen                                                                 | Peers screen                                                               | Settings screen                                                                  | Menubar/taskbar                                                                       |
+| Files screen                                                               | Explore screen                                                                 | Peers screen                                                               | Settings screen                                                                  | Menubar: top right                                                                      |
```
Comment (johnnymatthews) [inline docs/install/ipfs-desktop.md:16]:
```suggestion
| Files screen                                                               | Explore screen                                                                 | Peers screen                                                               | Settings screen                                                                  | Menubar/taskbar                                                                      |
```

This is OS-specific. We don't want to do that.

File: docs/install/ipfs-desktop.md [hunk 2]
```diff
@@ -13,16 +13,16 @@ If you already have an IPFS node on your computer, IPFS Desktop will act as a co
...
 | ![Screenshot of the Files screen](./images/ipfs-desktop/desktop-files.png) | ![Screenshot of the Explore screen](./images/ipfs-desktop/desktop-explore.png) | ![Screenshot of the Peers screen](./images/ipfs-desktop/desktop-peers.png) | ![Screenshot of the Settings screen](./images/ipfs-desktop/desktop-settings.png) | ![Screenshot of Mac/Windows menus](./images/ipfs-desktop/desktop-menubar-taskbar.png) |
 
 ### Feature highlights
 
-- **Start your node at system startup (Mac/Windows) and control it from your OS** using the convenient menubar/system tray menu.
+- **Start your node at system startup (Mac/Windows) and control it from your OS** by right-clicking the IPFS icon on the top right of your computer's menubar. <!-- ![IPFS Menubar Icon](.images/ipfs-menubar-icon.png). How do I get an image added? Is this the right format for an inline image? -->
```
Comment (johnnymatthews) [inline docs/install/ipfs-desktop.md:22]:
```suggestion
- **Start your node at system startup (Mac/Windows) and control it from your OS** using the convenient menubar/system tray menu.
```

Again, the location of the IPFS icon is specific to each operating system. We can expect users to know where to look if we say _status bar icon_ or something like that. Revert this back to the original.

File: docs/install/ipfs-desktop.md [hunk 3]
```diff
@@ -13,16 +13,16 @@ If you already have an IPFS node on your computer, IPFS Desktop will act as a co
...
 - **Quickly import files, folders, and screenshots to IPFS** in a variety of convenient ways, including drag-and-drop and (for Windows) right-clicking a file/folder's icon.
 - **Easily manage the contents of your node** with a familiar file browser that offers quick shortcuts for renaming/moving/pinning files and folders, previewing many common file formats directly in IPFS Desktop, copying content IDs or shareable links to your clipboard, and more.
-- **Quick download for CIDs, IPFS paths, and IPNS paths** — choose `Download...` from your menubar/status tray, paste in a hash, and you're good to go.
+- **Quick download for CIDs, IPFS paths, and IPNS paths** — choose `Download...` by right-clicking the IPFS icon on your computer's menubar, paste in a hash, and you're good to go.
```
Comment (johnnymatthews) [inline docs/install/ipfs-desktop.md:25]: This is good!

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
I've requested some changes. Just go through these comments @Annamarie2019 and commit my suggestions, then we're good to go with merging this.

**Comment by Annamarie2019** [comment]
...so I've committed all changes, so I believe my work is done here, yes?

**Review by johnnymatthews — APPROVED** [review]

**Pull request merged**